package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "webpack-demo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "html-webpack-plugin": "^5.5.3",
+    "webpack": "^5.4.0",
+    "webpack-cli": "^4.2.0"
+  },
```

Comment:
Kindly include a start script to enable a user to run your project as your project isn't running using the localhost(npm start isn't starting your project), also remember to include this step in your README file 😄 . When you implement this change , the next reviewer can then proceed with your review

Reply:
fixed the issues